### DISCUSSION TITLE
Poetry script execution based on project.toml script section when installing or updating the project.

### DISCUSSION THREAD

**Discussion by provision95** [python-poetry/poetry#6161] [Q&A] [answered]
Hello, I wonder if there is already such functionality, which would allow poetry to automatically run scripts defined by .toml without executing it with `poetry run script` command and instead running it seamlessly when installing or updating the project? If this functionality does not exist yet, what do you think about adding it to features section?

**Comment by lsorber** [comment, marked as answer]
https://github.com/nat-n/poethepoet

**Reply by provision95** [reply]
Thank You, I do not know how I missed this tool, have a nice day:)
